libpod/define/info.go

```diff
@@ -9,6 +9,7 @@ import (
 // running libpod/podman
 // swagger:model LibpodInfo
 type Info struct {
+	Client     *ClientInfo            `json:"client"`
```

Comment:
This is a API change and nothing is ever set via API so this makes little sense to expose this over the wire.

I recommend to add a new type on the client side only, something like this in cmd/podman/system/info.go:
```
type Info struct {
	*define.Info
	Client *ClientInfo `json:"client"`
}

type ClientInfo struct {
	OSArch   string `json:"OS"`
	Provider string `json:"provider"`
	Version  string `json:"version"`
}
```

Reply:
ok, fair enough ... i kind of figured that was coming.